chamge/types.h

```diff
@@ -23,6 +23,12 @@ typedef enum {
   CHAMGE_RETURN_ASYNC,
 } ChamgeReturn;
 
+typedef enum {
+  CHAMGE_ERROR_AMQP = 1,
+  CHAMGE_ERROR_NODE,
+  CHAMGE_ERROR_BACKEND,
+} ChamgeError;
```

Comment:
According to the above declaration of errors, this enum can be;

```suggestion:-4+0
typedef enum {
  CHAMGE_BACKEND_ERROR_INVALID_PARAMETER,
  CHAMGE_BACKEND_ERROR_INACCASSIBLE,
  CHAMGE_BACKEND_ERROR_OPERATION_FAILURE,
} ChamgeBackendError;
```
But, the naming is up to you.
Surely, declaring quarks should be in a separated commit.